src/sss_client/common.c

```diff
@@ -1144,7 +1128,26 @@ errno_t sss_strnlen(const char *str, size_t maxlen, size_t *len)
 }
 
 #if HAVE_PTHREAD
-typedef void (*sss_mutex_init)(void);
+bool sss_is_lockfree_mode(void)
+{
+    const char *env = NULL;
+    static enum {
+            MODE_UNDEF,
+            MODE_LOCKING,
+            MODE_LOCKFREE
+        } mode = MODE_UNDEF;
+
+    if (mode == MODE_UNDEF) {
+        env = getenv("SSS_NSS_LOCKFREE");
+        if ((env != NULL) && (strcmp(env, "NO") == 0)) {
```

Comment:
strcasecmp?

Reply:
Hm, right, for `SSS_NSS_USE_MEMCACHE` we use `strcasecmp()`.
I'll update this.

Reply:
And for `_SSS_LOOPS` we use `strcmp()`.
A bit inconsistent...

Reply:
Changed.